docs/_content/standaard/standaard.md

```diff
@@ -141,6 +142,57 @@ foutbericht, waarbij de `invalid-params` key meer informatie bevat over de fout.
 Indien een parameter wordt toegepast die niet in de OAS van de betreffende API
 staat, dan MOET de API antwoorden met een HTTP 400 foutbericht.
 
+## Notificaties
+
+Componenten dienen events te publiceren naar (een) notificatiecomponent(en)
+(NC). De NC MOET volledig de
+[`openapi.yaml`](../../../api-specificatie/nc/openapi.yaml) implementeren.
+
+### Kanalen
+
+Een producer van events, zoals ZRC, DRC, BRC, ZTC MOETEN hun kanaal registreren
+bij de NC indien dit nog niet bestaat. De volgende kanalen worden op dit moment
+onderkend:
+
+* ZRC: `zaken`
```

Comment:
Ik zou niet voor elke bron dit (kanalen), en kenmerken verderop, opnemen in dit document maar aangeven dat het in de documentatie van het component MOET staan beschreven welk kanaal/kenmerken geldig zijn.